GNUmakefile

```
@@ -195,7 +195,7 @@ HASHSUM_PROGS := \
 
 $(info Detected OS = $(OS))
 
-ifneq ($(OS),Windows_NT)
+ifeq (,$(findstring MINGW,$(OS)))
```

```suggestion
ifeq (,$(findstring _NT,$(OS)))
```
For MSVC (outside of MinGW).

---

It seems we cannot support MSVC by this way because `Detected OS = MSYS_NT-10.0-20348`...